modules/programs/waybar.nix

```
@@ -0,0 +1,378 @@
+{ config, lib, pkgs, ... }:
+
```

Can add
```suggestion
with lib;
```
here to avoid having to prefix so many function calls by `lib.`.

---

Thanks